lib/v2/udn/breaking-news.js

```diff
@@ -0,0 +1,66 @@
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const link = `/news/breaknews/1/${id}#breaknews`;
+    const name = await getLinkName(ctx, link);
+    const rootUrl = 'https://udn.com/';
+    const response = await got(`${rootUrl}/api/more?page=1&channelId=1&cate_id=${id}&type=breaknews`);
+    const items = await Promise.all(
+        response.data.lists.map(async (item) => {
+            const link = item.titleLink.indexOf('http://') === 0 || item.titleLink.indexOf('https://') === 0 ? item.titleLink : `${rootUrl}${item.titleLink}`;
+
+            const { author, description } = await ctx.cache.tryGet(link, async () => {
+                const result = await got(link);
+                const $ = cheerio.load(result.data);
+
+                return {
+                    author: $('.article-content__author a').text().trim(),
+                    description: art(path.join(__dirname, 'templates/description.art'), {
+                        description: $('.article-content__editor  p').text().trim(),
```

Comment:
```suggestion
                        description: $('.article-content__editor').html(),
```
Why parse text only? Some inline pictures will be skipped in that way like [this one](https://udn.com/news/story/7270/6575991).

Please add `$('.article-body__editor').html()` also in order to parse part of the content that blocked behind paywall like [this one](https://udn.com/news/story/6812/6576003).

Reply:
Should we cleanup the style element from the HTML output?